pkg/rbd/rbd_util.go

```diff
@@ -227,24 +240,80 @@ func execCommand(command string, args []string) ([]byte, error) {
 	return cmd.CombinedOutput()
 }
 
+func getMonsAndClusterID(options map[string]string) (monitors, clusterID, monInSecret string, err error) {
+	var ok bool
+
+	monitors, ok = options["monitors"]
+	if !ok {
+		// if mons are not set in options, check if they are set in secret
+		if monInSecret, ok = options["monValueFromSecret"]; !ok {
+			// if mons are not in secret, check if we have a cluster-id
+			if clusterID, ok = options["clusterID"]; !ok {
+				err = errors.New("either monitors or monValueFromSecret or clusterID must be set")
```

Comment:
do we need to log `err` here?

Reply:
Carrying over from prior implementation [here](https://github.com/ceph/ceph-csi/pull/241/files#diff-8e27516879efe971bf4755fadf585e06L241). I do not see a need, let me know if there is a specific reason you want it logged that I am missing.

Reply:
yeah my bad you are right